doc/examples/tracking_bootstrap_peaks.py

```diff
@@ -76,24 +68,26 @@
                                           affine, step_size=.5)
 streamlines = Streamlines(boot_streamline_generator)
 
-renderer.clear()
-renderer.add(actor.line(streamlines, cmap.line_colors(streamlines)))
-window.record(renderer, out_path='bootstrap_dg_CSD.png', size=(600, 600))
+save_trk("tractogram_bootstrap_dg.trk", streamlines, affine, labels.shape)
+
+if have_fury:
+    r = window.Renderer()
+    r.add(actor.line(streamlines, colormap.line_colors(streamlines)))
+    window.record(r, out_path='tractogram_bootstrap_dg.png', size=(800, 800))
+    if interactive:
+        window.show(r)
 
 """
-.. figure:: bootstrap_dg_CSD.png
+.. figure:: tractogram_bootstrap_dg.png
    :align: center
 
    **Corpus Callosum Bootstrap Probabilistic Direction Getter**
 
 We have created a bootstrapped probabilistic set of streamlines. If you repeat
 the fiber tracking (keeping all inputs the same) you will NOT get exactly the
-same set of streamlines. We can save the streamlines as a Trackvis file so it
-can be loaded into other software for visualization or further analysis.
+same set of streamlines.
 """
 
-save_trk("bootstrap_dg_CSD.trk", streamlines, affine, labels.shape)
-
 """
 Example #2: Closest peak direction getter with CSD Model
```

Comment:
Can you put this text in bold or as a subtitle like this:
```
Example #2: Closest peak direction getter with CSD Model
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Can you do the same for example 1 above? thank you

Reply:
done.